### ISSUE TITLE
space or special characters or double extension

### ISSUE THREAD

**Issue by andromeda979** [KITPraktomatTeam/Praktomat#224]
How can I fix the name of a file written by a student in a wrong way?
I get an error by uploading a file with space or special characters or double extension as "**test for2.cx.c**"

I want to create a control function on the file if the file is incorrectly written

Is there already a solution?

**Comment by mhecker** [comment]
Hi! Thanks for reporting the problem! 

Can you please tell us the exact error message you get when uploading such a file?
Thanks!

**Comment by andromeda979** [comment]
```
 C - Compiler : failed

gcc: fatal error: no input files
compilation terminated.
```

**Comment by mhecker** [comment]
Please confirm that the issue is indeed fixed by fbdd5fb
Also, make sure that in the task configuration, the file_pattern does indeed support filenames containing spaces, e.g.: 
`^[a-zA-Z0-9_` ]*\.[cC]$`
